src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ProjectConfigurationFileChangeDetector.cs

```diff
@@ -128,14 +118,18 @@ protected virtual void OnInitializationFinished()
     // Protected virtual for testing
     protected virtual IEnumerable<string> GetExistingConfigurationFiles(string workspaceDirectory)
     {
-        var files = DirectoryHelper.GetFilteredFiles(workspaceDirectory, _languageServerFeatureOptions.ProjectConfigurationFileName, s_ignoredDirectories, logger: _logger);
+        using var _ = _logger.BeginScope("Searching for existing project configuration files");
```

Comment:
Probably want to merge https://github.com/dotnet/razor/pull/8233 first or cherry-pick it onto this PR.

It was silly of me to write it that way at the time just because we weren't using Scopes yet.

Reply:
Also of note though, most of the loggers NoOp for `BeginScope`, so this might not produce the useful info you imagine (though it will be nice to "come online" if we complete https://github.com/dotnet/razor/issues/8232).